code/modules/research/xenobiology/crossbreeding/stabilized.dm

```diff
@@ -84,10 +84,10 @@ Stabilized extracts:
 
 /obj/item/slimecross/stabilized/pyrite
 	colour = "pyrite"
-
+/*
```

Comment:
Not needed.

Reply:
I was told having it in your back removed your slowdown. Which is the reason we were yeeting the red slime extra across the board?

Reply:
Yet you left in the bluespace extract?

Reply:
The bluespace extract when in a backpack, teleports someone randomly within sight from my understanding.
It isn't an actively usable item like the regenerative bluespace extract where you can use it on someone to teleport them.
If you would like, I can comment that out as well. But It wasn't contributing to the same problem that the other one was being commented out for, like the stabilized red extract is.